lib/Steps/Integrations/ReactNative.ts

```diff
@@ -135,6 +140,41 @@ export class ReactNative extends MobileProject {
 
     await Promise.all(promises);
 
+    let host: string | null = null
+    try {
+      host = (new URL(this.url || '')).host;
+    } catch (_error) {
+      // ignore
+    }
+    const orgSlag = _.get(answers, 'config.organization.slug', null);
+    const projectId = _.get(answers, 'config.project.id', null);
+    const projectIssuesUrl = host && orgSlag && projectId
+      ? `https://${orgSlag}.${host}/issues/?project=${projectId}`
+      : null;
+
+    l(`
+To make sure everything is set up correctly, put the following code snippet into your application.
+The snippet will create a button that, when tapped, sends a test event to Sentry.
+`);
+
+    if (projectIssuesUrl) {
```

Comment:
`m`: maybe better to move this below the code snippet

Reply:
I did it like that at first, but the code snippet disappears in the text. 
Like this, it stands out more, in modern terminals the link is clickable so that doesn't get lost in the text.